### PR TITLE
Expanding consultancy page(s)

### PR DESCRIPTION
**Pull request by NPDebs**
This PR addresses issue #678.
Following the request to add more consultancy services to the website, a new index page is now under construction.

**Changes made:**
1. Created a `/consulting` subfolder to house all the individual services offered by OLS.
2. Added an `index.md` file to the newly created subfolder which will be the new landing page.
3. Stored the information about services in a `consulting.yaml` file.
4. Displayed the stored information on small cards in the index page (i.e /consulting). On these cards are links to individual services.
5. Used separate _service-x.md_ files for more consultancy pages (with url `/consulting/ally-skills`).

Screenshot (573)

**Questions/Comment for reviewer(s):**
1. Is there any info missing from the cards that should be added?
2. Currently, the same hero image is used across all the pages. Should each page have its unique photo that best captures the service?
3. Finally, if Patri's PR #679 is merged first (for the sake of urgency), it will be added to the correct directory in this PR. 

Thanks!

### CODE REVIEW

**Comment by NPDebs** [comment]
(Non-urgent)  Nuding @bebatut and @yochannah. 👈
Only review when you have the time. 🙏

**Comment by bebatut** [comment]
Thanks @NPDebs.
The structure looks globally good. I have just few comments
1. I am not sure how many consultancy page we will get and if having them using the `consultancy.yml` is not too complicated
2. If we have quite some, we might want to organize them in groups

But nothing really bad just some thoughts. @yochannah what do you think?

**Comment by yochannah** [comment]
I think this is prettymuch what @pherterich and I discussed last week, so it looks good to me! Also tagging @malvikasharan as I think she was discussing this with Patricia maybe?

File: _data/consulting.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+- name: Ally Skills
+  url: /consulting/ally-skills.html
+  description:  |
+        Multiple OLS community members are trained Ally skill facilitators. 
+        We offer ad-hoc in-house workshops (starting at £2,000 GBP for universities) 
+        as well as anyone-may-register workshops...
+
+- name: Accessibilty Consulting
```
Comment (yochannah) [inline _data/consulting.yaml:9]:
```suggestion
- name: Video call accessibility
```
Typo fix + narrowing down the scope - I don't want to assert we know all of accessibility!

**Comment by yochannah** [comment]
Adding another note - we're likely to want to link to individual service pages in the future (for potential clients), so having them separate but indexed on the main page will be very helpful.

**Comment by NPDebs** [comment]
> 1. I am not sure how many consultancy page we will get and if having them using the `consultancy.yml` is not too complicated

I tried to keep the structure as basic as possible, so that future contributors won't struggle. I'm not sure how many there will be. Probably why you asked Yo. 😅🏃🏽‍♀️

File: _data/consulting.yaml (hunk 1 above)
Comment (NPDebs) [reply _data/consulting.yaml]: Oops. Thank you!

And I know what you mean.

**Review by yochannah — APPROVED** [review]
Approved! note that the organisational leadership card doesn't link anywhere - is that my fault? 😬

**Comment by NPDebs** [comment]
> Approved! note that the organisational leadership card doesn't link anywhere - is that my fault? 😬 

Thanks for merging, @yochannah! 

Nice catch. It didn't initially link anywhere because the _organisational leadership_ page was yet to be merged. Now that both PRs have been merged, there are two options:

1. Link the leadership page and open a new PR.
2. Revert the merge on this PR, add the changes here and we merge again.

I'm fine with either 1 or 2, really.

**Comment by bebatut** [comment]
Go for 1. Quick :)

**Comment by NPDebs** [comment]
> Go for 1. Quick :)

Done. Ready at PR #699...

**Pull request merged**